templates/cloud/index.html

```diff
@@ -32,12 +32,19 @@
 
 <section class="p-strip--light p-strip--hero is-deep is-bordered">
   <div class="row">
-    <div class="col-10">
+    <div class="col-7">
       <h1>云计算的标准操作系统</h1>
       <p class="p-heading--four">70％的公有云工作负载和54％的OpenStack云<a href="#fn1" id="r1">*</a></p>
       <p><a href="https://www.ubuntu.com/download/cloud" class="p-button--positive">安装OpenStack</a></p>
-      <p>如您已明确企业的需求，可直接<a href="https://pages.ubuntu.com/contact-us-in-chinese.html" class="p-link--external">联系我们</a>获得技术支持以及认证服务。</p>
+      <br>
+      <p>如您已明确企业的需求，可直接<a href="https://pages.ubuntu.com/contact-us-in-chinese.html" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - cloud', 'eventValue' : undefined });">联系我们</a>获得技术支持以及认证服务。</p>     
     </div>
+
+    <div class="col-5">
+      <img src="https://assets.ubuntu.com/v1/17a7067f-451bib.jpg">
+      <p>打破私有云经济学的迷思，Canonical的托管私有云优于公有云。<a href="https://pages.ubuntu.com/Multi-cloudEbook_chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'ebook', 'eventLabel' : 'cloud-ebook', 'eventValue' : undefined });">查看分析机构451 Research的研究报告。</a></p>
```

Comment:
External link missing the external class

Reply:
fixed all the missing external classes, will push soon.